src/explicitode.jl

```diff
@@ -242,6 +242,47 @@ function taylorstep!(f!, t::Taylor1{T}, x::Vector{Taylor1{U}},
     return δt
 end
 
+
+
+"""
+    _determine_parsing!(parse_eqs::Bool, f, t, x, params)
+    _determine_parsing!(parse_eqs::Bool, f, t, x, dx, params)
+
+Check if the parsed method of `jetcoeffs!` exists and check it
+runs without error.
+"""
+function _determine_parsing!(parse_eqs::Bool, f, t, x, params)
+    parse_eqs = parse_eqs &&
+        !isempty(methodswith(Val{f}, TaylorIntegration.jetcoeffs!))
+    if parse_eqs
+        try
+            jetcoeffs!(Val(f), t, x, params)
+        catch
+            @warn("""Unable to use the parsed method of `jetcoeffs!`
+            despite of having `parse_eqs=true`, due to some internal error.
+            Using `parse_eqs = $false`""")
+            parse_eqs = false
+        end
+    end
+    return parse_eqs
+end
+function _determine_parsing!(parse_eqs::Bool, f, t, x, dx, params)
+    parse_eqs = parse_eqs &&
+        !isempty(methodswith(Val{f}, TaylorIntegration.jetcoeffs!))
```

Comment:
Nice! Using `methodswith` this way, we only look for `jetcoeffs!` methods related to `f`, right?

Reply:
Yes! I think this is much better than before